driver-scala/src/it/scala/org/mongodb/scala/RequiresMongoDBISpec.scala

```diff
@@ -16,6 +16,7 @@
 
 package org.mongodb.scala
 
+import com.mongodb.ClusterFixture.getServerApi
 import com.mongodb.MongoClientSettings
```

Comment:
If you change this import to `org.mongodb.scala.MongoClientSettings` you wont need to set the codec registry.

Reply:
Done